include/wx/aui/auibar.h

```diff
@@ -255,7 +273,9 @@ class WXDLLIMPEXP_AUI wxAuiToolBarItem
 };
 
 #ifndef SWIG
+// wxAuiToolBarItemArray is deprecated but kept for backwards compatibility
 WX_DECLARE_USER_EXPORTED_OBJARRAY(wxAuiToolBarItem, wxAuiToolBarItemArray, WXDLLIMPEXP_AUI);
+WX_DECLARE_EXPORTED_LIST(wxAuiToolBarItem, wxAuiToolBarItemList);
```

Comment:
Let's avoid using the macro-based containers for the new code, i.e. please just use `std::vector<wxAuiToolBarItem*>` or maybe `std::vector<std::unique_ptr<wxAuiToolBarItem>>` if it should own the items.

Reply:
So new containers should all be standard library containers now?

Even though the toolbar should own the items, I don't think using the `unique_ptr` is going to work, since we have to pass this object down to the toolbar art provider to draw the dropdown list. A pure vector with the toolbar itself doing the memory management would work.

Reply:
> So new containers should all be standard library containers now?

Yes, there is no reason not to use them any more as we require C++11 and this means we can count on them being available.